supreme/src/jvmTest/kotlin/at/asitplus/signum/supreme/crypt/Test.kt

```diff
@@ -0,0 +1,106 @@
+import at.asitplus.signum.indispensable.Ciphertext
+import at.asitplus.signum.indispensable.SymmetricEncryptionAlgorithm
+import at.asitplus.signum.supreme.crypt.decrypt
+import at.asitplus.signum.supreme.crypt.encryptorFor
+import at.asitplus.signum.supreme.crypt.randomIV
+import at.asitplus.signum.supreme.crypt.randomKey
+import io.kotest.core.spec.style.FreeSpec
+import io.kotest.datatest.withData
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.types.shouldBeInstanceOf
+import javax.crypto.Cipher
+import javax.crypto.spec.GCMParameterSpec
+import javax.crypto.spec.IvParameterSpec
+import javax.crypto.spec.SecretKeySpec
+import kotlin.random.Random
+
+@ExperimentalStdlibApi
+class JvmAESTest : FreeSpec({
+
+    "Against JCA" - {
+        "Illegal IV Size" - {
+            withData(
+                SymmetricEncryptionAlgorithm.AES_128.CBC.PLAIN,
+                SymmetricEncryptionAlgorithm.AES_192.CBC.PLAIN,
+                SymmetricEncryptionAlgorithm.AES_256.CBC.PLAIN,
+
+                //JVM knows no AES-CBC-HMAC
+
+                SymmetricEncryptionAlgorithm.AES_128.GCM,
```

Comment:
Isn't that covered in `commonTest` already?

Reply:
one level of nesting too many. it does not check against illegal iv sizes, just against JCA. i removed the confusing label